src/physics/PhysicsList.cc

```diff
@@ -33,6 +40,55 @@ void PhysicsList::ConstructProcess() {
   AddParameterization();
   Shielding::ConstructProcess();
   ConstructOpticalProcesses();
+  EnableThermalNeutronScattering();
+}
+
+// Add thermal neutron scattering to this physics list.
+// Based on the technique shown here: http://tinyurl.com/hszjtln
```

Comment:
Good point. I used tinyurl links because they're word-wrap friendly, but the originals should be more trustworthy.